app/jobs/github_languages_job.rb

```diff
@@ -0,0 +1,28 @@
+require 'octokit'
+
+module GithubLanguagesJob
+  extend self
+
+  def client
+    credentials = {
+        client_id: Settings.github.client_id,
+        client_secret: Settings.github.client_secret
+    }
+    @client ||= Octokit::Client.new(credentials)
+  end
+
+  def run
+    Project.with_github_url.each do |p|
+      begin
+        languages = client.languages("#{p.github_repo_name}/#{p.github_repo_user_name}")
+        language_names = languages.to_hash.keys
+        existing_languages = p.languages.collect(&:name).collect(&:to_sym)
+        incoming_languages = language_names - existing_languages
+        incoming_languages.each { |language| p.languages.build(name: language) }
```

Comment:
it looks like this should only create languages once in the Language table ... - in my local system I have many repeats - maybe from accidentally re-running the job, but I wonder here if we should put an index on the language table name to ensure uniqueness, and possibly re-structure this code to use `Language.find_or_create_by name: language`